pkg/cluster/cluster.go

```diff
@@ -94,7 +93,6 @@ type manager struct {
 	denyAssignments          authorization.DenyAssignmentClient
 	fpPrivateEndpoints       network.PrivateEndpointsClient // TODO: use armFPPrivateEndpoints instead.
 	armFPPrivateEndpoints    armnetwork.PrivateEndpointsClient
-	rpPrivateLinkServices    network.PrivateLinkServicesClient // TODO: use armRPPrivateLinkServices instead.
```

Comment:
Can this be removed too? /pkg/util/azureclient/mgmt/network/privatelinkservices.go

Reply:
Not yet. pkg/util/purge/purge.go still uses the old one.